makefile_lib

```diff
@@ -47,19 +47,19 @@ LST:=$(SRC_C:.c=.lst)
 INC:=-I$(INC_DIR) -I$(SRC_DIR) -I$(RES_DIR)
 
 # default flags
-DEF_FLAGS_M68K:=-m68000 -Wall -fno-builtin $(INC)
+DEF_FLAGS_M68K:=-m68000 -Wall -Wextra -Wno-shift-negative-value -fno-builtin $(INC)
 DEF_FLAGS_Z80:=-i$(SRC_DIR) -i$(INC_DIR)
 
 release: envcheck
-release: FLAGS:=$(DEF_FLAGS_M68K) -O3 -fno-web -fno-gcse -fno-unit-at-a-time -fomit-frame-pointer
+release: FLAGS:=$(DEF_FLAGS_M68K) -O3 -fuse-linker-plugin -fno-lto -fno-web -fno-gcse -fno-unit-at-a-time -fomit-frame-pointer -flto
 release: $(SGDK)/lib/libmd.a
 
 debug: envcheck
 debug: FLAGS:=$(DEF_FLAGS_M68K) -O1 -ggdb -DDEBUG=1
 debug: $(SGDK)/lib/libmd_debug.a
 
 asm: envcheck
-asm: FLAGS:=$(DEF_FLAGS_M68K) -O3 -fno-web -fno-gcse -fno-unit-at-a-time -fomit-frame-pointer -S
+asm: FLAGS:=$(DEF_FLAGS_M68K) -O3  -fuse-linker-plugin -fno-lto -fno-web -fno-gcse -fno-unit-at-a-time -fomit-frame-pointer -S
```

Comment:
As above. What is the benefit of LTO here?

Reply:
You are also right, there is no point in having this fno-lto here.